src/index.html

```diff
@@ -3,12 +3,55 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#home" class="img"><img src="images/logo.png" alt="moyo logo"></a>
+
+
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="a width is-active color box">APPLE</a></li>
+          <li class="nav__item"><a href="#samsung" class="a color">SAMSUNG</a></li>
+          <li class="nav__item"><a href="#smartphones" class="a color">SMARTPHONES</a></li>
+          <li class="nav__item" data-qa="hover"><a href="#laptops" class="a color">LAPTOPS & COMPUTERS</a></li>
+          <li class="nav__item"><a href="#gadgets" class="a color">GADGETS</a></li>
+          <li class="nav__item"><a href="#tablets" class="a color">TABLETS</a></li>
+          <li class="nav__item"><a href="#photo" class="a color">PHOTO</a></li>
+          <li class="nav__item"><a href="#video" class="a color">VIDEO</a></li>
+        </ul>
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
```suggestion
    <header class="header">
      <a href="#home" class="img">
        <img src="images/logo.png" alt="moyo logo">
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="a width is-active color box">
              APPLE
            </a>
          </li>
          <li class="nav__item"><a href="#samsung" class="a color">SAMSUNG</a></li>
          <li class="nav__item"><a href="#smartphones" class="a color">SMARTPHONES</a></li>
          <li class="nav__item" data-qa="hover"><a href="#laptops" class="a color">LAPTOPS & COMPUTERS</a></li>
          <li class="nav__item"><a href="#gadgets" class="a color">GADGETS</a></li>
          <li class="nav__item"><a href="#tablets" class="a color">TABLETS</a></li>
          <li class="nav__item"><a href="#photo" class="a color">PHOTO</a></li>
          <li class="nav__item"><a href="#video" class="a color">VIDEO</a></li>
        </ul>
```
add some formating, it's hard to read, also don't add that many spaces in the end and between elements, it's unnecessary